app/webpacker/components/Panel/pages/EditPersonPage/EditPersonForm.jsx

```diff
@@ -100,20 +98,32 @@ function EditPersonForm({ wcaId, clearWcaId, setResponse }) {
 
   return (
     <>
-      <Item>
-        <Item.Content>
-          <Item.Header>
-            WCA ID:
-            {' '}
-            {person.wca_id}
-          </Item.Header>
-          <Item.Description>
-            <Button onClick={() => clearWcaId()}>
-              Clear
-            </Button>
-          </Item.Description>
-        </Item.Content>
-      </Item>
+      {response != null && (
+        <Message
+          success={response.success}
+          error={!response.success}
+          content={response.message}
+        >
+          <Message.Content>
+            {response.success && (
+            <>
+              Success!
+              <br />
+            </>
```

Comment:
Should be indented, same with the the fragment below.
```suggestion
              <>
                Success!
                <br />
              </>
```

Reply:
Thanks Kevin, applied the change.